src/pymodaq_plugins_thorlabs/daq_move_plugins/daq_move_KPZ101.py

```diff
@@ -0,0 +1,133 @@
+    _epsilon = 0.01
+    data_actuator_type = DataActuatorType.DataActuator
+    params = [
+                 {'title': 'Serial Number:', 'name': 'serial_number', 'type': 'list',
+                  'limits': serialnumbers_piezo, 'value': serialnumbers_piezo[0]}
+
+             ] + comon_parameters_fun(is_multiaxes, axes_names=_axes_names, epsilon=_epsilon)
+
+    def ini_attributes(self):
+        self.controller: Piezo = None
+        self._move_done = False
+
+    def get_actuator_value(self):
+        """Get the current value from the hardware with scaling conversion.
+
+        Returns
+        -------
+        float: The position obtained after scaling conversion.
+        """
+        pos = DataActuator(
+            data=self.controller.get_position(),
+            units=self.controller.get_units()
+        )
+        pos = self.get_position_with_scaling(pos)
+        return pos
+
+    def close(self):
+        """Terminate the communication protocol"""
+        if self.is_master:
+            self.controller.close()
+
+    def commit_settings(self, param: Parameter):
+        """Apply the consequences of a change of value in the detector settings
+
+        Parameters
+        ----------
+        param: Parameter
+            A given parameter (within detector_settings) whose value has been changed by the user
+        """
+        if param.name() == 'axis':
+            self.axis_unit = self.controller.get_units(self.axis_value)
+
+    def ini_stage(self, controller=None):
+        """Actuator communication initialization
+
+        Parameters
+        ----------
+        controller: (object)
+            custom object of a PyMoDAQ plugin (Slave case). None if only one actuator by controller (Master case)
+
+        Returns
+        -------
+        info: str
+        initialized: bool
+            False if initialization failed otherwise True
+        """
+
+        if self.is_master:
+            self.controller = Piezo()
+            self.controller.connect(self.settings['serial_number'])
+        else:
+            self.controller = controller
+
+        self.axis_unit = self.controller.get_units(self.axis_value)
+
+        info = f'{self.controller.name} - {self.controller.serial_number}'
+        initialized = True
+        return info, initialized
+
+    def move_abs(self, value: DataActuator):
+        """ Move the actuator to the absolute target defined by value
+
+        Parameters
+        ----------
+        value: (float) value of the absolute target positioning
+        """
+        self._move_done = False
+        value = self.check_bound(value)
+        self.target_value = value
+        value = self.set_position_with_scaling(value) 
+        self.controller.move_abs(value.value())
+
+    def move_rel(self, value: DataActuator):
+        """ Move the actuator to the relative target actuator value defined by value
+
+        Parameters
+        ----------
+        value: (float) value of the relative target positioning
```

Comment:
replace the float type to DataActuator

Reply:
All the concerns are addressed based on your feedback.